src/components/Home.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="restaurants">
+    <div class="restaurants__img"></div>
+    <h1 class="restaurants__header">Najlepsze restauracje w Poznaniu</h1>
+    <ul class="restaurants__list">
+      <li
+        class="restaurants__item"
+        v-for="restaurant in restaurants"
+        :key="
+     restaurant.restaurant.id"
+      >
+        <a :href="restaurant.restaurant.events_url">{{restaurant.restaurant.name}}</a>
+        Ocena: {{restaurant.restaurant.user_rating.aggregate_rating}}
+        Kuchnia: {{restaurant.restaurant.cuisines}}
+        <router-link :to="`/restaurant/${restaurant.restaurant.id}`">Więcej</router-link>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+import store from '../store'
+
+export default {
+  name: 'Home',
+  computed: {
+    restaurants () {
+      if (store.restaurants.length > 0) {
```

Comment:
Checking for special cases is a very good practice. Early-return in such a case so that the "main" code flow isn't indented is an even better one. While it's certainly much more wide-spread in languages such as Go, it works very well with JavaScript too.